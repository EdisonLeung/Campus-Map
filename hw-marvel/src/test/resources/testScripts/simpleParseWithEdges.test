# Tests file parsing with nodes and edges

LoadGraph simpleParse graphWithEdge.csv

# list nodes and children
ListNodes simpleParse
ListChildren simpleParse a
ListChildren simpleParse b