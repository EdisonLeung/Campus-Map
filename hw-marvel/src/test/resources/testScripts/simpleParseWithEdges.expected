# Tests file parsing with nodes and edges

loaded graph simpleParse

# list nodes and children
simpleParse contains: a b
the children of a in simpleParse are: b(AB)
the children of b in simpleParse are: a(AB)